swagger.yaml

```diff
@@ -39,27 +50,56 @@ parameters:
     description: If true, JSON response will be pretty-printed
 
 definitions:
-  Info:
+  response:
+    type: object
+    properties:
+      links:
+        $ref: "#/definitions/Links"
+      data:
+        $ref: "#/definitions/UserData"
+
+  Links:
+    type: object
+
+  UserData:
+    type: object
     properties:
-      name:
+      id:
         type: string
-        description: Name of the API.
-      time:
+        example: "kuok"
+      type:
         type: string
-        description: Build time in ISO 8601 format.
-      unixTime:
-        type: integer
-        description: Build time as milliseconds since the unix epoch.
-      commit:
+        example: "spendingauthority"
```

Comment:
Instead of using `example`, can you use [`enum`](https://github.com/osu-mist/express-api-skeleton/blob/master/openapi.yaml#L151-L153) to represent the resource type?